### PR TITLE
add WithFont function

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this?

/kind cleanup
/kind feature

#### What this PR does / why we need it:

follow up of https://github.com/kubernetes-sigs/release-utils/pull/44#discussion_r836157061

add a new function called `WithFont` that return the version function with a selected ASCII font

/assign @saschagrunert @puerco 
cc @kubernetes-sigs/release-engineering 

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
add WithFont function
```

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
That's good! 

/hold for @puerco

**Comment by puerco** [comment]
You just made a lot of maintainers happy further down the stream, thanks friends :)
/lgtm
/hold cancel

**Review by xmudrii — APPROVED** [review]

**Pull request merged**
